Documentation/ceph-object-multisite.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Object Multisite
+weight: 2250
+indent: true
+---
+# NOTE
+
+At the moment there is no realm pull feature implemented for the ceph-object-realm CR at the moment. Thus syncing objects between Ceph clusters is not working yet. Completing this feature is going to be completed by the Rook 1.4 release.
+
+# Object Multisite
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
+
+Multisite also allows object stores to be independent and isolated from other object stores in a cluster.
+
+When a ceph-object-store is created without the `zone` section; a realm, zone group, and zone is created with the same name as the ceph-object-store.
+
+Since it is the only ceph-object-store in the realm, the data in the ceph-object-store remain independent and isolated from others on the same cluster.
+
+When a ceph-object-store is created with the `zone` section, the ceph-object-store will join a custom created zone, zone group, and realm with a different than its own.
+
+This allows the ceph-object-store to replicate its data over multiple Ceph clusters.
+
+To review core multisite concepts please read the [ceph-multisite design overview](/design/ceph/object/ceph-multisite-overview.md).
+
+## Prerequisites
+
+This guide assumes a Rook cluster as explained in the [Quickstart](ceph-quickstart.md).
+
+# Creating Object Multisite
+
+If an admin wants to set up multisite on a Rook Ceph cluster, the admin should create:
+
+    - A [realm](/Documentation/ceph-object-multisite-crd.md)
+    - A [zonegroup](/Documentation/ceph-object-multisite-crd.md)
+    - A [zone](/Documentation/ceph-object-multisite-crd.md)
+    - An [object-store](/Documentation/ceph-object-store.md) with the zone section
+
+object-multisite.yaml in the [examples](/cluster/examples/kubernetes/ceph/) directory can be used to create the multisite CRDs.
+```console
+$ kubectl create -f object-multisite.yaml
```

Comment:
Can you remove all the dollar signs? It's easier to copy and paste the full line this way. Thanks.